### PR TITLE
Update plan.md with Migration Note

### PR DESCRIPTION
**Pull request by lbdyck**
The one thing missing that I just realized is there was nothing about migrating the CBT environment, or any other environment (if there are), to the new box.

### CODE REVIEW

File: programs/infrastructure/plan.md [hunk 1]
```diff
@@ -5,9 +5,6 @@ title: Plan
 
 # Mainframe Infrastructure Plan for Open Mainframe Project
 
-* TOC
-{:toc}
-
```
Comment (jmertic) [inline programs/infrastructure/plan.md:10]: @lbdyck Can you revert this change? This adds the automatic TOC.

Comment (lbdyck) [reply programs/infrastructure/plan.md:10]: I don't know how - can you accept it and then add that back in?

**Comment by lbdyck** [comment]
I updated and restored the toc
